Assets/Scripts/Player/PlayerController.cs

```diff
@@ -262,6 +264,24 @@ public class PlayerController : MonoBehaviour {
 				crawlingSound.Stop();
 			}
 		}
+
+		if (isGrounded && transformation.State == TransformationState.Wolf && Mathf.Abs(velocity.x) > 0.1f) {
+			if (!wolfRunningSound.isPlaying) {
+				wolfRunningSound.Play();
+			}
+		}
+		else {
+			wolfRunningSound.Stop();
+		}
+
+		if (isGrounded && transformation.State == TransformationState.Human && Mathf.Abs(velocity.x) > 0.1f) {
+			if (!humanRunningSound.isPlaying) {
+				humanRunningSound.Play();
+			}
+		}
+		else {
+			humanRunningSound.Stop();
+		}
```

Comment:
If you start transforming the sound doesn't stop. This is due to returning the function above.